pages/_app.js

```diff
@@ -1,32 +1,79 @@
-import Script from 'next/script'
-import '../styles/globals.css'
-import * as gtag from '../lib/gtag'
+import '../styles/globals.css';
+import * as gtag from '../lib/gtag';
+import Head from 'next/head';
+import { useEffect } from 'react';
+import Script from 'next/script';
 
 function MyApp({ Component, pageProps }) {
-  return (
-    <>
-      <Script
-        strategy="afterInteractive"
-        src={`https://www.googletagmanager.com/gtag/js?id=${gtag.GA_TRACKING_ID}`}
-      />
-      <Script
-        id="gtag-init"
-        strategy="afterInteractive"
-        dangerouslySetInnerHTML={{
-          __html: `
+  const apiKey = process.env.NEXT_PUBLIC_COOKIE_CONTROL_KEY;
+
+  useEffect(() => {
+    loadCookieControl(apiKey);
```

Comment:
Sorry if my last comment wasn't clear - this isn't necessary if we're loading the script with a <script> tag, since we only need to run the code once!